specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/virtualWan.json

```diff
@@ -3927,6 +4125,130 @@
+    },
+    "VirtualWanVpnProfileParameters": {
+      "properties": {
+        "vpnServerConfigurationResourceId": {
+          "type": "string",
+          "description": "VpnServerConfiguration partial resource uri with which VirtualWan is associated to."
+        },
+        "authenticationMethod": {
+          "$ref": "./network.json#/definitions/AuthenticationMethod",
+          "description": "VPN client authentication method."
+        }	
+      },
+      "description": "Virtual Wan Vpn profile parameters Vpn profile generation."
+    },
+    "P2SVpnConnectionHealthRequest": {
+      "properties": {
+        "vpnUserNamesFilter": {
+          "type": "array",
+          "items": {
+            "type": "string",
+			"description": "P2S vpn user name."
+          },
+          "description": "The list of p2s vpn user names whose p2s vpn connection detailed health to retrieve for."
+        },
+        "outputBlobSasUrl": {
+          "type": "string",
+          "description": "The sas-url to download the P2S Vpn connection health detail."
+        }	
+      },
+      "description": "List of P2S Vpn connection health request."
+    },
+    "P2SVpnConnectionHealth": {
+      "properties": {
+        "sasUrl": {
+          "type": "string",
+          "description": "Returned sas url of the blob to which the p2s vpn connection detailed health will be written. "
+        }
+      },
+      "description": "P2S Vpn connection detailed health written to sas url."
+    },
+    "P2SConnectionConfigurationProperties": {
+      "properties": {
+        "vpnClientAddressPool": {
+          "$ref": "./virtualNetwork.json#/definitions/AddressSpace",
+          "description": "The reference of the address space resource which represents Address space for P2S VpnClient."
+        },
+        "provisioningState": {
+          "$ref": "./network.json#/definitions/ProvisioningState",
+          "description": "The provisioning state of the P2SConnectionConfiguration resource."
+        }
+      },
+      "description": "Parameters for P2SConnectionConfiguration."
+    },
+    "P2SConnectionConfiguration": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/P2SConnectionConfigurationProperties",
+          "description": "Properties of the P2S connection configuration."
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the resource that is unique within a resource group. This name can be used to access the resource."
+        },
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "P2SConnectionConfiguration Resource."
+    },
+    "DefaultError": {
```

Comment:
@Nilambari @majastrz this is a fourth (!) definition of the same cloud error in Network specs. We're planning to fix that soon by re-using ARM's common definitions but for now please reference `CloudError` from `ddosCustomPolicy.json`

Reply:
Sure. I will refer to CloudError from ddosCustomPolicy.json everywhere in this swagger spec.